### PR TITLE
[tf-builder] Added kExprI64Clz, kExprI64Ctz, and kExprI64Popcnt to the tf-builder.

### PR DESCRIPTION
**Pull request by gahaas**

### CODE REVIEW

File: test/cctest/wasm/test-run-wasm.cc [hunk 1]
```diff
@@ -765,6 +765,171 @@ TEST(Run_WasmInt64Binops) {
...
+    WasmRunner<int32_t> r(kMachInt64);
+    // return op a
+    BUILD(r, WASM_UNOP(opcode, WASM_GET_LOCAL(0)));
+    CHECK_EQ(expected, r.Call(a));
+  }
+}
+
+
+TEST(Run_WasmInt64Clz) {
+  TestInt64ToInt32Unop(kExprI64Clz, 0, 0x8000100000000000);
```
Comment (titzer) [inline test/cctest/wasm/test-run-wasm.cc]: These build a new graph, do a fresh compile, and run each time. Can you refactor this code to build and compile once?

Comment (gahaas) [inline test/cctest/wasm/test-run-wasm.cc]: Done.

**Comment by titzer** [comment]
lgtm

**Comment by binji** [comment]
Do these return i32? I'm getting this failure running the spec tests:
`WasmModule::Instantiate(): Compiling WASM function #13:clz failed:Result = ImplicitReturn[0] expected type int64, found ExprI64Clz of type int32 @+3`

**Comment by gahaas** [comment]
Yes, I64CLZ, I64CTZ, and I64POPCNT all return i32.

**Pull request merged**